the-name-card/src/components/WidgetItem/Items/Link.tsx

```diff
@@ -0,0 +1,53 @@
+import { z } from 'zod';
+
+import { WidgetProp } from '.';
+import './Link.scss';
+
+const LinkValues = z.object({ description: z.string(), link: z.string() });
+
+export type LinkValues = z.infer<typeof LinkValues>;
+
+export const Link: React.FC<WidgetProp> = ({ value, updateValue }) => {
+	const rawValue = JSON.parse(value);
+	const linkValues = LinkValues.parse(rawValue);
```

Comment:
this might result in the page crashing because there's no error handler if `JSON.parse` or `LinkValues.parse` fails

Reply:
yeap, will be working on the error handling for this